src/coordinates.jl

```diff
@@ -0,0 +1,172 @@
+struct CoordinateSystem{S} end
+
+"""
+    UnkownSpace
+"""
+const UnkownSpace = CoordinateSystem{:unkown}()
```

Comment:
I assume you mean
```suggestion
const UnknownSpace = CoordinateSystem{:unknown}()
```
And the same in the docstring.